flask_webapp/static/js/menu_list.js

```diff
@@ -36,8 +36,8 @@ function clearChildNodes(elt_id){
   scan_dev_row.querySelectorAll('*').forEach(n => n.remove());
 }
 
-function drawNoDeviceMessage(){
-  var no_dev_text = document.createTextNode("Couldn't Find any bluetooth devices.");
+/* function drawNoDeviceMessage(){

```

Comment:
@fsjames either move these commented functions to the bottom of the file or remove them all together if they aren't being used.

Reply:
actually i see the new function implementations in the bottom so might as well just remove these old ones.